File: C/ReverseNumber.c
#include<stdio.h>
#include<conio.h>
int main()
{
    int x,y=0,r;
    printf("Enter a number : ");
    scanf("%d",&x);
    while(x != 0){
        r = x%10;
        y = y*10+r;
        x = x/10;
    }
    printf("Reverse number is : %d",y);
    getch();
}

File: ReverseNumber.java
import java.util.*;
class ReverseNumber
{
    public static void main(String[]args)
    {
        int rev=0,n=0;
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter the number: ");
        n=sc.nextInt();
        sc.close();
        while(n!=0)
        {
            int temp=n%10;
                rev=(rev*10)+temp;
                n=n/10;
        }
        System.out.println("Reverse are: "+rev);
    }
}
